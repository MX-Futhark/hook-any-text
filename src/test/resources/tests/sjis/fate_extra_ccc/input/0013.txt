96 D8 91 A2 82 CC 8A F7 81 41 96 D8 96 DA 82 CC 83 5E 83 43 83 8B 81 41 83 8C 83 67 83 8D 82 C8 91 8B 98 67 81 42 0A 81 63 81 63 96 AD 82 C8 89 F9 82 A9 82 B5 82 B3 82 C9 8B B9 82 F0 92 F7 82 DF 95 74 82 AF 82 E7 82 EA 82 E9 81 42 0A 8D A1 82 CD 90 BC 97 EF 32 30 33 30 94 4E 81 5C 81 5C 81 5C 92 6E 8F E3 82 C9 82 A2 82 E9 8E A9 95 AA 82 C5 82 CD 8C 6F 8C B1 82 B5 82 E6 82 A4 82 CC 82 C8 82 A2 81 41 0A 8B 8C 82 A2 8E 9E 91 E3 82 CC 23 52 55 42 53 82 DC 82 C8 82 D1 82 E2 23 52 55 42 45 8A 77 8E C9 23 52 45 4E 44 82 CC 95 97 8C 69 82 AA 8D 4C 82 AA 82 C1 82 C4 82 A2 82 E9 81 42 00 00 00 00 00 00 00 01 00 00 00 02 00 00 00 03 00 00 00 04 00 00 00 05 00 00 00 06 00 00 00 07 00 00 00 08 00 00 00 09 00 00 00 0A 00 00 00 0B 00 00 00 0C 00 00 00 0D 00 00 00 0E 00 00 00 0F 00 00 00 10 00 00 00 11 00 00 00 12 00 00 00 13 00 00 00 14 00 00 00 15 00 00 00 16 00 00 00 17 00 00 00 18 00 00 00 19 00 00 00 1A 00 00 00 1B 00 00 00 1C 00 00 00 1D 00 00 00 1E 00 00 00 1F 00 00 00 20 00 00 00 21 00 00 00 22 00 00 00 23 00 00 00 24 00 00 00 25 00 00 00 26 00 00 00 27 00 00 00 28 00 00 00 29 00 00 00 2A 00 00 00 2B 00 00 00 2C 00 00 00 2D 00 00 00 2E 00 00 00 2F 00 00 00 30 00 00 00 31 00 00 00 32 00 00 00 33 00 00 00 34 00 00 00 35 00 00 00 36 00 00 00 37 00 00 00 38 00 00 00 39 00 00 00 3A 00 00 00 3B 00 00 00 3C 00 00 00 3D 00 00 00 3E 00 00 00 3F 00 00 00 40 00 00 00 41 00 00 00 42 00 00 00 43 00 00 00 44 00 00 00 45 00 00 00 46 00 00 00 47 00 00 00 48 00 00 00 49 00 00 00 4A 00 00 00 4B 00 00 00 4C 00 00 00 4D 00 00 00 4E 00 00 00 4F 00 00 00 50 00 00 00 51 00 00 00 52 00 00 00 53 00 00 00 54 00 00 00 55 00 00 0B 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 1F 01 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 91 43 00 00 BE 42 00 00 00 00 00 00 60 41 00 00 00 00 FF FF FF FF 00 00 98 43 00 00 BE 42 00 00 00 00 00 00 00 00 00 00 60 41 FF FF FF FF 00 00 91 43 00 00 DA 42 00 00 00 00 00 00 60 41 00 00 60 41 FF FF FF FF 00 00 98 43 00 00 DA 42 00 00 00 00 7C 19 A3 48 02 00 00 00 14 16 C9 08 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF 00 00 00 00 44 18 A3 48 02 00 00 00 34 16 C9 08 00 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF 00 01 00 00 AC 18 A3 48 02 00 00 00 54 16 C9 08 00 00 00 00 00 00 00 00 00 00 00 00