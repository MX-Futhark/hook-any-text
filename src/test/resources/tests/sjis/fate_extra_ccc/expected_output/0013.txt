["木造の机、木目のタイル、レトロな窓枠。
……妙な懐かしさに胸を締め付けられる。
今は西暦2030年———地上にいる自分では経験しようのない、
旧い時代の学舎の風景が広がっている。"]